hdfs/src/test/java/io/ddf/hdfs/HDFSDDFManagerTests.java

```diff
@@ -26,6 +27,10 @@ public void testDDFConfig() throws Exception {
     public static void startServer() throws Exception {
         LOG = LoggerFactory.getLogger(HDFSDDFManagerTests.class);
         manager = new HDFSDDFManager(System.getenv("HDFS_URI"));
+        try {
+            new HDFSDDFManager("invalidpath");
+            assert false;
+        } catch (DDFException e) {}
```

Comment:
Please move this to another test, because if this fails, the whole class won't run
Test failed because of it throws UnknownHostException